<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CdnPurger</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1,5,1,8,1">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>네이버 클라우드 CDN Purger</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="cdnInstanceNo"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>cdnInstanceNo : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>isWholeDomain : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="radioButton_Yes">
             <property name="text">
              <string>Yes</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_IsWholeDomain</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_No">
             <property name="text">
              <string>No</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_IsWholeDomain</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>isWholePurge : </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QRadioButton" name="radioButton_Whole">
             <property name="text">
              <string>Whole</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_isWholePurge</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_Directory">
             <property name="text">
              <string>Directory</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_isWholePurge</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_Files">
             <property name="text">
              <string>Files</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_isWholePurge</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Purge할 도메인명을 입력하세요.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="domainEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="labelDescription">
      <property name="text">
       <string>Purge할 파일 경로명을 입력하세요.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pathEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QPushButton" name="okBtn">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetBtn">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="action_KeySetting"/>
    <addaction name="action_fileMenu"/>
   </widget>
   <widget class="QMenu" name="versionMenu">
    <property name="title">
     <string>버전</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="versionMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_fileMenu">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_KeySetting">
   <property name="text">
    <string>Key Setting</string>
   </property>
   <property name="toolTip">
    <string>Key Setting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_isWholePurge"/>
  <buttongroup name="buttonGroup_IsWholeDomain"/>
 </buttongroups>
</ui>
